### PR TITLE
Update gce.md

### PR DESCRIPTION
**Pull request by wackxu**
fix bad url

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4858)

### CODE REVIEW

File: docs/getting-started-guides/gce.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ wget -q -O - https://get.k8s.io | bash
 
 Once this command completes, you will have a master VM and four worker VMs, running as a Kubernetes cluster.
 
-By default, some containers will already be running on your cluster. Containers like `fluentd` provide [logging](/docs/user-guide/logging/overview), while `heapster` provides [monitoring](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/cluster-monitoring/README.md) services.
+By default, some containers will already be running on your cluster. Containers like `fluentd` provide [logging](https://kubernetes.io/docs/concepts/cluster-administration/logging/), while `heapster` provides [monitoring](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/cluster-monitoring/README.md) services.
```
Comment (chenopis) [inline docs/getting-started-guides/gce.md]: We should use relative URLs, so the logging link should be: `[logging](/docs/concepts/cluster-administration/logging/)`

Comment (wackxu) [inline docs/getting-started-guides/gce.md]: ok

**Comment by chenopis** [comment]
@wackxu Why did you close this PR? Looks like you were able to update the URL correctly.

**Comment by wackxu** [comment]
@chenopis I am sorry and I have reopen it.

**Comment by chenopis** [comment]
@wackxu No worries. Thx!

**Pull request merged**